src/lib/booleans.js

```diff
@@ -0,0 +1,80 @@
+function negate(a) {
+  return !a;
+};
+
+function both(a, b) {
+  return a && b;
+};
+
+function either(a, b) {
+  return a || b;
+};
+
+function none(a, b) {
+  return !a && !b; 
+};
+
+function one(a, b) {
+  return (a && !b) || (!a && b);
+};
+
+function truthiness(a) {
+  return !!a;
+};
+
+function isEqual(a, b) {
+  return a === b;
+};
+
+function isGreaterThan(a, b) {
+  return a > b;
+};
+
+function isLessThanOrEqualTo(a, b) {
+  return a <= b;
+};
+
+function isOdd(a) {
+  return a % 2 !== 0;
+};
+
+function isEven(a) {
+  return a % 2 === 0;
+};
+
+function isSquare(a) {
+  return Number.isInteger(Math.sqrt(a));
+};
+
+function startsWith(char, string) {
+  return string.startsWith(char);
+};
+
+function containsVowels(string) {
+  if (string.match(/[aeiou]/gi)) {
+    return true;
+  }
+    return false;
```

Comment:
You can tweak this to make it a one liner. The `.match` function in the way youre using it here will either return the first instance it finds, or `null` if it finds no matches, so we can turn this into a boolean using the double negation trick (!!) that you are using for the truthiness function above:
```
  return !!string.match(/[a|e|i|o|u|]/i, 0);
```
Here if the result is a match, we dont want to return the value but a `true` boolean, so we can use a single ! to turn that truthy value false, then another to flip it back to true as we want.
If there is no match it will return `null` (a falsy value but _not strictly equal to false_), by the same logic we can flip it to truthy and back to falsy and give us the result we want :)